NAMESPACE

```diff
@@ -10,7 +10,11 @@ export("col_footnotes<-")
 export("col_info<-")
 export("col_total<-")
 export("colcount_format<-")
+export("colcount_na_str<-")
+export("colcount_visible<-")
 export("content_table<-")
+export("facet_colcount<-")
+export("facet_colcounts_visible<-")
```

Comment:
why not also an accessor? it should be relatively "free". not having it is a bit confusing to me

Reply:
because its not a pure setter. This setter goes to the specified path, then *loops through the children and sets each of their count visibility to `value`*. 

I don't really see a strong usecase for a getter with similar behavior.